--- tvtime-1.0.11/configure.ac~	2025-08-25 16:40:11.000000000 +0200
+++ tvtime-1.0.11/configure.ac	2025-08-25 16:40:47.343409849 +0200
@@ -39,7 +39,7 @@ AC_CHECK_FUNCS([asprintf atexit fork exe
 
 # Check for gettext
 AM_GNU_GETTEXT_VERSION
-AM_GNU_GETTEXT
+AM_GNU_GETTEXT([external])
 
 dnl ---------------------------------------------
 dnl zlib
@@ -243,7 +243,6 @@ AC_CONFIG_FILES([
 	data/Makefile
 	plugins/Makefile
 	src/Makefile
-	intl/Makefile
 	po/Makefile.in
 	m4/Makefile
 	docs/man/Makefile
--- tvtime-1.0.11/src/get_media_devices.c~	2016-02-24 16:25:31.000000000 +0100
+++ tvtime-1.0.11/src/get_media_devices.c	2025-08-25 17:50:15.998022265 +0200
@@ -23,6 +23,7 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <string.h>
 #include <stdlib.h>
 #include <malloc.h>
