--- tvtime-0.9.13/plugins/greedyh.asm	2004-08-31 01:52:05.000000000 +0000
+++ tvtime-0.9.13-new/plugins/greedyh.asm	2004-09-26 08:58:25.189617016 +0000
@@ -108,7 +108,7 @@
 #endif
 
              // save XBX (-fPIC)
-       //      ""PUSHX" %%"XBX"\n\t"
+             ""PUSHX" %%"XBX"\n\t"
 
              ""MOVX"  "_L1",          %%"XAX"\n\t"
              ""LEAX"  8(%%"XAX"),       %%"XBX"\n\t"      // next qword needed by DJR
@@ -257,7 +257,7 @@
              "jmp     1b\n\t"
              
              "1:\n\t"
-         //    ""POPX" %%"XBX"\n\t"
+             ""POPX" %%"XBX"\n\t"
 
              : /* no outputs */
 
@@ -277,7 +277,7 @@
                "m"(LoopCtr),
                "m"(QW256)
 
-             : ""XAX"", ""XBX"", ""XCX"", ""XDX"", ""XSI"", ""XDI"",
+             : ""XAX"", ""XCX"", ""XDX"", ""XSI"", ""XDI"",
 #ifdef ARCH_386
 // is this really necessary?
                "st", "st(1)", "st(2)", "st(3)", "st(4)", "st(5)", "st(6)", "st(7)",
Files tvtime-0.9.13/plugins/tomsmocomp/.SearchLoopBottom.inc.swp and tvtime-0.9.13-new/plugins/tomsmocomp/.SearchLoopBottom.inc.swp differ
diff -urN tvtime-0.9.13/plugins/tomsmocomp/SearchLoopBottom.inc tvtime-0.9.13-new/plugins/tomsmocomp/SearchLoopBottom.inc
--- tvtime-0.9.13/plugins/tomsmocomp/SearchLoopBottom.inc	2004-08-31 01:52:05.000000000 +0000
+++ tvtime-0.9.13-new/plugins/tomsmocomp/SearchLoopBottom.inc	2004-09-26 08:58:25.188617168 +0000
@@ -73,7 +73,7 @@
            "jb      1b\n\t"                    // y
 #endif
 
-    //        ""POPX" %%"XBX"\n\t"
+           ""POPX" %%"XBX"\n\t"
 
         : /* no outputs */
         : "m"(pBob),
@@ -95,7 +95,7 @@
           "m"(Max_Mov),
           "m"(YMask)
 
-        : ""XAX"", ""XBX"", ""XCX"", ""XDX"", ""XSI"", ""XDI"",
+        : ""XAX"", ""XCX"", ""XDX"", ""XSI"", ""XDI"",
           "st", "st(1)", "st(2)", "st(3)", "st(4)", "st(5)", "st(6)", "st(7)",
           "mm0", "mm1", "mm2", "mm3", "mm4", "mm5", "mm6", "mm7",
           "memory", "cc"
diff -urN tvtime-0.9.13/plugins/tomsmocomp/SearchLoopTop.inc tvtime-0.9.13-new/plugins/tomsmocomp/SearchLoopTop.inc
--- tvtime-0.9.13/plugins/tomsmocomp/SearchLoopTop.inc	2004-08-31 01:52:06.000000000 +0000
+++ tvtime-0.9.13-new/plugins/tomsmocomp/SearchLoopTop.inc	2004-09-26 08:58:25.186617472 +0000
@@ -104,7 +104,7 @@
              // XSI - next weave pixels, 1 line up
 
              // Save "XBX" (-fPIC)
-            // ""PUSHX" %%"XBX"\n\t"
+             ""PUSHX" %%"XBX"\n\t"
              
 #ifdef IS_SSE2
              
