--- tvtime-1.0.10/src/speedy.c.orig	2016-02-24 10:56:25.000000000 +0100
+++ tvtime-1.0.10/src/speedy.c	2016-07-04 20:30:52.212076878 +0200
@@ -547,7 +547,7 @@
 #ifdef ARCH_X86
 static void convert_uyvy_to_yuyv_scanline_mmx( uint8_t *uyvy_buf, uint8_t *yuyv_buf, int width )
 {
-#ifdef ARCH_386
+#if defined(ARCH_386) || defined(__ILP32__)
     __asm__ __volatile__(
         "   movl      %0, %%esi         \n"
         "   movl      %1, %%edi         \n"
@@ -575,8 +575,7 @@
         /* output */            :
         /* input */             : "g" (uyvy_buf), "g" (yuyv_buf), "g" (width)
         /* clobber registers */ : "cc", "edx", "esi", "edi" );
-#endif
-#ifdef ARCH_X86_64
+#elif defined(ARCH_X86_64)
     __asm__ __volatile__(
         "   movq      %0, %%rsi         \n"
         "   movq      %1, %%rdi         \n"
